### ISSUE TITLE
v1.25.0 Hyperkit missing `/grub/grub.cfg`

### ISSUE THREAD

**Issue by klaases** [kubernetes/minikube#13413]
### What Happened?

When running v1.25.0 on Mac using Hyperkit, the following file is missing `/grub/grub.cfg` from the Buildroot configuration.

### Attach the log file

```
😿  Failed to start hyperkit VM. Running "minikube delete" may fix it: creating host: create: Error creating machine: Error in driver during machine creation: extracting kernel: unable to find file /grub/grub.cfg

❌  Exiting due to GUEST_PROVISION: Failed to start host: creating host: create: Error creating machine: Error in driver during machine creation: extracting kernel: unable to find file /grub/grub.cfg
```

### Operating System

macOS (Default)

### Driver

HyperKit

**Comment by th3fallen** [comment]
same here

**Comment by afbjorklund** [comment]
The file looks unused, both isolinux.cfg (and grub.cfg)

**Comment by klaases** [comment]
Hi @afbjorklund, yes, this line was removed. However, we will still need the grub.cfg file to exist somewhere in the ISO in order to boot into UEFI.

Closing this out as the issue was resolved in release v1.25.1.
